sdk/storage/Azure.Storage.Blobs/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Added support for Encryption Scope SAS.
 - Added support for Encryption Scopes with BlobBaseClient.SyncCopyFromUriAsync().
 - Added support for generating SAS URLs with the Permanent Delete ('y') SAS permission.
+- Fixed regression from v12.9.0-beta.2, where BlobContainerClient.GetBlobs() and .GetBlobsByHierarchyAsync() was not parsing the Object Replication Metadata correctly
```

Comment:
Do need need to mention when the regression was introduced?  I don't know if we have ever done that before.

Reply:
I just wanted to note it somewhere so that people know where the regression exists in case they want to avoid it, but maybe the ChangeLog isn't the right place since it's supposed to note changes not where bugs or regression exist.